src/davinci/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.2.8 on 2021-12-12 19:55
+# Generated by Django 3.2.8 on 2021-12-19 19:20
```

Comment:
Maybe my comment is more applicable here: I would expect the changes in this file to come from a new migration script instead of amending it to the current one, since django keeps track of which migrations it has performed by the app name and file name alone. There is a risk with an existing DB that the changes here will not be applied to it because it has already executed the script name before